Categories/NSXMLElement+XMPP.m

```diff
@@ -120,58 +123,60 @@ - (NSArray *)elementsForXmlnsPrefix:(NSString *)nsPrefix
-		// Consider the following XML fragment:
-		// 
-		// <query xmlns="jabber:iq:private">
-		//   <x xmlns="some:other:namespace"></x>
-		// </query>
-		// 
-		// Calling [query elementsForName:@"x"] results in an empty array!
-		// 
-		// However, it will work properly if you use the following:
-		// [query elementsForLocalName:@"x" URI:@"some:other:namespace"]
-		// 
-		// The trouble with this is that we may not always know the xmlns in advance,
-		// so in this particular case there is no way to access the element without looping through the children.
-		// 
-		// This bug was submitted to apple on June 1st, 2007 and was classified as "serious".
-		// 
-		// --!!-- This bug does NOT exist in DDXML --!!--
-		
-		return nil;
-	}
-}
-
-/**
- * This method returns the first child element for the given name and given xmlns (as an NSXMLElement).
- * If no child elements exist for the given name and given xmlns, nil is returned.
-**/
-- (NSXMLElement *)elementForName:(NSString *)name xmlns:(NSString *)xmlns
-{
-	NSArray *elements = [self elementsForLocalName:name URI:xmlns];
-	if ([elements count] > 0)
-	{
-		return elements[0];
-	}
-	else
-	{
-		return nil;
-	}
-}
+  * This method returns the first child element for the given name (as an NSXMLElement).
+  * If no child elements exist for the given name, nil is returned.
+ **/
+ - (NSXMLElement *)elementForName:(NSString *)name
+ {
+     NSArray *elements = [self elementsForName:name];
+     if ([elements count] > 0)
+     {
+         return elements[0];
+     }
+     else
+     {
+         // There is a bug in the NSXMLElement elementsForName: method.
+         // Consider the following XML fragment:
+         //
+         // <query xmlns="jabber:iq:private">
+         //   <x xmlns="some:other:namespace"></x>
+         // </query>
+         //
+         // Calling [query elementsForName:@"x"] results in an empty array!
+         //
+         // However, it will work properly if you use the following:
+         // [query elementsForLocalName:@"x" URI:@"some:other:namespace"]
+         //
+         // The trouble with this is that we may not always know the xmlns in advance,
+         // so in this particular case there is no way to access the element without looping through the children.
+         //
+         // This bug was submitted to apple on June 1st, 2007 and was classified as "serious".
+         //
+         // --!!-- This bug does NOT exist in DDXML --!!--
+
+         return nil;
+     }
+ }
+
+ /**
+  * This method returns the first child element for the given name and given xmlns (as an NSXMLElement).
+  * If no child elements exist for the given name and given xmlns, nil is returned.
+ **/
+ - (NSXMLElement *)elementForName:(NSString *)name xmlns:(NSString *)xmlns
+ {
+     NSArray *elements = [self elementsForLocalName:name URI:xmlns];
+     if ([elements count] > 0)
+     {
+         return elements[0];
+     }
+     else
+     {
+         return nil;
+     }
+ }
+#endif
```

Comment:
It doesn't look like anything was changed here other than whitespace. If possible please revert the whitespace changes to minimize the diff